src/main/java/Duke.java

```diff
@@ -34,28 +32,50 @@ static void printFarewellMessage() {
         String farewell = "Bye. Hope you will complete everything for today!";
         System.out.println(farewell);
     }
-    static void recordTask(String phrase) {
```

Comment:
is this change really necessary for coding standards? seems more redundant after the change